inlineplz/linters/todolint.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import re
+import dirtyjson as json
+from inlineplz.decorators import linter
+from inlineplz.parsers.base import ParserBase
+
+
+@linter(
+    {
+        "name": "todolint",
+        "url": "https://bitbucket.org/atlassian/todolint",
+        "language": "all",
+        "patterns": ["*.*"],
+        "install": [["go", "get", "-u", "bitbucket.org/atlassian/todolint"]],
+        "help": ["todolint", "--help"],
+        "run": ["todolint"],
+        "rundefault": ["todolint"],
+        "dotfiles": [],
+        "autorun": True,
+        "run_per_file": False,
+    }
+)
+class TodoLintParser(ParserBase):
+    """Parse todolint output."""
+
+    def parse(self, lint_data):
+        messages = set()
+        lines = re.compile(r"\r?\n").split(lint_data)
+        line_pat = re.compile(r"^(.+?) \((\d+)\): (.+)$")
+
+        for line in lines:
+            m = line_pat.search(line)
```

Comment:
[inline-plz]: `prospector: pylint: Variable name "m" doesn't conform to snake_case naming style (invalid-name)`